roles/openshift_hosted/tasks/registry/registry.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Retrieve list of openshift nodes matching registry selector
+  command: >
+    {{ openshift.common.client_binary }} --api-version='v1' -o json
+    get nodes -n default --config={{ openshift_hosted_kubeconfig }}
+    --selector={{ openshift.hosted.registry.selector | default('') }}
```

Comment:
This would go to openshift_hosted_registry_selector if my above proposal is sane.